src/content/docs/paper/dev/api/recipes.md

```diff
@@ -7,7 +7,7 @@ slug: paper/dev/recipes
 Recipes are a way to define a way to craft a particular item. They are defined by a plugin or
 datapack, however we are only going to cover the plugin side of things here.
 
-## [`ShapedRecipe`](jd:paper:org.bukkit.inventory.ShapedRecipe)
+## [`ShapedRecipe`](jd:paper:org.bukkit.inventory.ShapedRecipeBROKERN)
```

Comment:
revert

Reply:
This was for test the url check but not works its ignored... Then not sure even if the plugin is running

Reply:
fixed